db/system_keyspace.cc

```diff
@@ -1763,8 +1764,14 @@ future<> system_keyspace::drop_truncation_rp_records() {
     auto rs = co_await execute_cql(req);
 
     bool any = false;
-    co_await coroutine::parallel_for_each(*rs, [&] (const cql3::untyped_result_set_row& row) -> future<> {
-        auto table_uuid = table_id(row.get_as<utils::UUID>("table_uuid"));
+    std::unordered_set<table_id> to_delete;
+    auto& db = _qp.db();
+    auto max_concurrency = std::min(1024u, smp::count * 8);
```

Comment:
Conflicts with https://github.com/scylladb/scylladb/pull/25678, right?
Let's turn this PR to draft until #25678 is promoted

Reply:
Yes.

Reply:
How does one do that, beyond call it RFC or similar?